internal/policies/apparmor/apparmor.go

```diff
@@ -110,15 +110,21 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 	if err := os.Rename(apparmorPath, oldApparmorPath); err != nil {
 		return fmt.Errorf(i18n.G("can't rename apparmor directory %q to %q: %v"), apparmorPath, oldApparmorPath, err)
 	}
-	defer cleanupApparmorDir(ctx, oldApparmorPath, apparmorPath)
+	defer cleanupOldApparmorDir(ctx, oldApparmorPath, apparmorPath)
 
 	// Dump assets to the adsys/machine/ subdirectory with correct ownership. If no assets is present while entries != nil, we want to return an error.
-	if err := assetsDumper(ctx, "scripts/apparmor/", apparmorPath, -1, -1); err != nil {
+	if err := assetsDumper(ctx, "apparmor/", apparmorPath, -1, -1); err != nil {
 		return err
 	}
 
+	idx := slices.IndexFunc(entries, func(e entry.Entry) bool { return e.Key == "apparmor-machine" })
+	if idx == -1 {
+		log.Warning(ctx, i18n.G("No valid entry found for the apparmor machine policy"))
+		return nil
+	}
+
+	e := entries[idx]
```

Comment:
ok, I think we can get it that way for now and same for users. It may not be future-proof, but as we expect always one key for machine, and one for users, this is ok.